# Relatório Parcial

## RESUMO

        A grafita é um bem mineral de importância tecnológica emergente com as novas propriedades descobertas nas últimas décadas no ramo da engenharia de nanomateriais, como no emprego de fabricação de baterias elétricas, supercondutores e fibras leves de alta resistência, e com potencial para fabricação de materiais essenciais para a indústria. Estes novos usos têm aumentado a demanda pela commoditie, trazendo assim, a necessidade de descoberta de novos depósitos economicamente viáveis levando em conta sua localização, volume e grau de pureza. Recentemente, as técnicas de aprendizagem de máquina têm aumentado a viabilidade dos projetos de prospecção mineral devido ao seu baixo custo de execução e sua alta capacidade de correlação de inúmeras variáveis simultaneamente.
        Com isto, neste projeto, pretende-se utilizar algoritmos de inteligência computacional e dados de sensores remotos para identificar padrões entre os atributos geofísicos e suas classes litológicas mineralizantes, bem como de suas ocorrências minerais. Assim, desenvolvendo novos mapas litológicos para confrontar os existentes e mapas prospectivos de minério de grafita no sistema de nappes de Socorro–Guaxupé no nordeste do estado de São Paulo, divisa com Minas Gerais.

## I. INTRODUÇÃO	

### 1.1 APRESENTAÇÃO

        A grafita possui grande importância industrial e sua demanda tem aumentado devido aos novos usos com potencial de crescimento ainda maior com a ascensão dos motores elétricos em detrimento dos motores à combustão e a necessidade de baterias de alta capacidade para aqueles. Assim, este material tem se tornado estratégico com os preços da grafita de alto teor (grafita em flocos) ultrapassando a casa dos milhares de dólares por toneladas métricas dependendo do tamanho dos flocos.

### 1.2 OBJETIVOS

        Esta pesquisa tem como objetivo a prospecção de áreas potenciais de mineralização de minério de grafita no sistema de nappes de Socorro–Guaxupé com o auxílio das técnicas de aprendizagem de máquina supervisionada e não-supervisionada com dados de sensores aeroportados e orbitais desenvolvendo mapas preditivos litológicos e prospectivo mineral. 

### 1.3 ÁREA DE ESTUDO

        O área de estudo tem como foco os terrenos granulíticos do nordeste de São Paulo, divisa com o estado de Minas Gerais, região em que as rochas sofreram um alto grau de metamorfismo possibilitando a mineralização de grafita. 

In [1]:
# imports
from src import Build_mc, Upload_litologia

import matplotlib.pyplot as plt
import geopandas as gpd

/home/ggrl/.config/ambiente_geologico/lib/python3.10/site-packages/geopandas/_compat.py:112: UserWarning: The Shapely GEOS version (3.10.2-CAPI-1.16.0) is incompatible with the GEOS version PyGEOS was compiled with (3.10.4-CAPI-1.16.2). Conversions between both will be slow.
  warnings.warn(


In [2]:
import warnings
warnings.filterwarnings("ignore")
%matplotlib widget
%load_ext autoreload
%reload_ext autoreload

In [4]:
quadricula = Build_mc(escala='250k', ID=['SF23'])

100%|█████████████████████████| 1/1 [00:00<00:00, 385.65it/s]
16it [00:00, 11620.58it/s]


In [5]:
Upload_litologia(quadricula,'litologia_1kk')

100%|███████████████████████| 16/16 [00:00<00:00, 129.17it/s]

 - litologia_1kk atualizado na folha: SF23_VA
 - litologia_1kk atualizado na folha: SF23_VC
 - litologia_1kk atualizado na folha: SF23_YA
 - litologia_1kk atualizado na folha: SF23_YC
 - litologia_1kk atualizado na folha: SF23_VB
 - litologia_1kk atualizado na folha: SF23_VD
 - litologia_1kk atualizado na folha: SF23_YB
 - litologia_1kk atualizado na folha: SF23_YD
 - litologia_1kk atualizado na folha: SF23_XA
 - litologia_1kk atualizado na folha: SF23_XC
 - litologia_1kk atualizado na folha: SF23_ZA
 - litologia_1kk atualizado na folha: SF23_ZC
 - litologia_1kk atualizado na folha: SF23_XB
 - litologia_1kk atualizado na folha: SF23_XD
 - litologia_1kk atualizado na folha: SF23_ZB
 - litologia_1kk atualizado na folha: SF23_ZD


In [ ]:
lista_folhas = list(quadricula.keys())
lista_folhas

['SF23_VA',
 'SF23_VC',
 'SF23_YA',
 'SF23_YC',
 'SF23_VB',
 'SF23_VD',
 'SF23_YB',
 'SF23_YD',
 'SF23_XA',
 'SF23_XC',
 'SF23_ZA',
 'SF23_ZC',
 'SF23_XB',
 'SF23_XD',
 'SF23_ZB',
 'SF23_ZD']

In [16]:
SF23_YB = quadricula['SF23_YB']

print(SF23_YB.keys())
print('')

print(SF23_YB['folha'])
print('')

SF23_YB['litologia_1kk'].explore('SIGLA',legend=False)

dict_keys(['folha', 'escala', 'litologia_1kk', 'litologia_100k', 'litologia_50k'])

EPSG                                                    32723
SIGLA                                                    None
geometry    POLYGON ((-46.5 -22, -45 -22, -45 -23, -46.5 -...
Name: SF23_YB, dtype: object



##  II. CONTEXTO GEOLÓGICO        

### 2.1 NAPPE SOCORRO

### 2.2 NAPPE GUAXUPÉ

 III. A GRAFITA	6
 
3.1 APRESENTAÇÃO	6

3.2 OCORRÊNCIA	6

3.3 TIPOS DE MINERALIZAÇÃO DE GRAFITA	6

3.4 GRAFITA NO SISTEMA DE NAPPES SOCORRO –GUAXUPÉ	6

 IV. MATERIAIS	7
 
4.1 APRESENTAÇÃO	7

4.2 PROJETOS AEROGEOFÍSICOS	7

4.2.1 SÃO JOSÉ DOS CAMPOS – RESENDE	7

4.2.2 SÃO PAULO – RIO DE JANEIRO (ÁREA SÃO PAULO)	7

4.2.3 ÁREA 14 MINAS GERAIS – POÇOS DE CALDAS – VARGINHA – ANDRELÂNDIA	7

4.3 FERRAMENTAS	7

 V. MÉTODOS	8
 
5.1 APRESENTAÇÃO	8

5.2 MÉTODO AEROMAGNETOMÉTRICO	8

5.3 PROCESSAMENTO DE DADOS AEROMAGNETOMÉTRICOS	8

 5.3.1 Pré-processamento dos dados aeromagnetométricos	8
 
 5.3.2 Interpolação dos dados aeromagnetométricos	8
 
 5.3.3 Amplitude do Sinal Analítico	8
 
 5.3.4 Gradiente Horizontal Total	8
 
5.4 MÉTODO AEROGAMAESPECTROMÉTRICO	8

5.5 PROCESSAMENTO DE DADOS AEROGAMAESPECTROMÉTRICOS	8

 5.5.1 Pré-processamento dos dados aerogamaespectrométricos	8
 
 5.5.2 Interpolação dos dados gama espectrométricos	8
 
 5.5.3 Mapas ternários	8
 

5.6 MÉTODOS DE CLASSIFICAÇÃO NÃO-SUPERVISIONADA	8

 5.6.1 SELF-ORGANIZING MAPS (SOM)	8
 
5.7 MÉTODOS DE CLASSIFICAÇÃO SUPERVISIONADA	8

 5.7.1 RANDOM FORESTS	8
 
 5.7.1 SUPPORT VECTOR MACHINES	8
 
VI. RESULTADOS E DISCUSSÕES	9

 6.1 APRESENTAÇÃO	9
 
 
 6.2 INTERPRETAÇÃO DOS DADOS AEROMAGNETOMÉTRICOS	9
 
VII. CONCLUSÕES	10

VIII. BIBLIOGRAFIA	11
